src/object.d

```diff
@@ -3257,8 +3252,6 @@ private size_t getArrayHash(in TypeInfo element, in void* ptr, in size_t count)
 }
 
 
-// @@@BUG5835@@@ tests:
-
```

Comment:
> This is not normal practice

But it is. We put bugzilla issue numbers next to corresponding unittests all the time.